server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -287,4 +289,28 @@ public void test_kepserver_regclass_cast_query() throws Exception {
             ""
         ));
     }
+
+    @Test
+    public void test_pg_constrains_conkey_array_populated() throws Exception {
+        execute("CREATE TABLE doc.tbl (" +
+            "not_null1 int not null," +
+            "int_col int," +
+            "long_col bigint," +
+            "str_col text," +
+            "not_null2 bigint not null," +
+            "checked int CONSTRAINT positive CHECK(checked>0)," +
+            "checked_and_not_null int not null CONSTRAINT check_and_not_null check (checked_and_not_null > 10)," +
+            "PRIMARY KEY (int_col, long_col, str_col)," +
+            "CONSTRAINT many_cols_and_functions CHECK(abs(not_null2) * abs(not_null1::bigint) > abs(int_col::bigint)))"
+        );
+        int reloid = OidHash.relationOid(OidHash.Type.TABLE, new RelationName("doc", "tbl"));
+        response = execute("select i.conkey, i.conname, i.contype from pg_catalog.pg_constraint i where i.conrelid  = " + reloid + " order by i.conname");
+        assertThat(printedTable(response.rows()), is(
```

Comment:
not related to conkey but also removed not-null constraints to align with PG. 
https://www.db-fiddle.com/f/h6SSAXRC1RYSC5oCrrmQYx/11
 
 From https://www.postgresql.org/docs/current/catalog-pg-constraint.html:
> Not-null constraints are represented in the [pg_attribute](https://www.postgresql.org/docs/current/catalog-pg-attribute.html) catalog, not here.